src/backend/joanie/core/filters/admin.py

```diff
@@ -56,9 +57,12 @@ class CourseRunAdminFilterSet(filters.FilterSet):
 
     class Meta:
         model = models.CourseRun
-        fields: List[str] = ["query"]
+        fields: List[str] = ["query", "start", "state"]
 
     query = filters.CharFilter(method="filter_by_query")
+    start = filters.IsoDateTimeFilter(method="filter_by_start")
```

Comment:
Do we really need to specify a method to filter ?
Furthemore, I wonder if by default, this filter allows to pass the comparator (lt, lte, gt or gte) inside the parameter 🤔